Cargo.toml

```diff
@@ -30,11 +30,13 @@ revm = { git = "https://github.com/risechain/revm", rev = "7b42abb672deacde9e053
 alloy-provider = "0.2.1"
 alloy-transport = "0.2.1"
 alloy-transport-http = "0.2.1"
+libmdbx = "0.5.0"
```

Comment:
Let's put this in its own `# On-Disk Storage dependencies` section with the same dedicated feature TODO.